doc/sphinx/addendum/type-classes.rst

```diff
@@ -295,14 +295,16 @@ the Existing Instance command to achieve the same effect.
 Summary of the commands
 -----------------------
 
-.. cmd:: Class @ident {? @binders} : {? @sort} := {? @ident} { {+; @ident :{? >} @term } }
+.. cmd:: Class @inductive_definition {* with @inductive_definition }
 
    The :cmd:`Class` command is used to declare a typeclass with parameters
    :token:`binders` and fields the declared record fields.
 
+   This command supports the :attr:`Cumulative`, :attr:`NonCumulative` and :attr:`Private` attributes.
```

Comment:
Similar to the question you raised in the Gallina chapter, this should either read "This commands supports the same attributes as the Inductive command" or it should be fixed to list the actual full list of attributes that this command supports.

Reply:
I listed all the attributes.  I have the same text for these commands:

```
finite_token: [
| "Inductive"
| "CoInductive"
| "Variant"
| "Record"
| "Structure"
| "Class"
]
```

It may be tedious to get all the attributes right and correct in the doc.  Eventually we should come up with a semi-automatic way to extract the relevant attributes for each command.  Probably means creating some common code to check attributes.

Reply:
Indeed, some future work in sight!